Make preprocessor token limit configurable (#5768)
verilator/verilator: docs/guide/exe_verilator.rst

   Disable printing the Verilation statistics report, see :ref:`Verilation
   Summary Report`.

.. option:: --preproc-token-limit

>    Verilator limits the number of tokens it can process on a single line to
>    prevent infinite loops and other hangs. This argument allows configuring
>    that limit. The default is 40000 tokens.

Reviewer: ```suggestion
   Rarely needed. Configure the limit of the number of tokens Verilator
   can process on a single line to prevent infinite loops and other hangs.
   Defaults to 40000 tokens.
```
